db/install.xml

```diff
@@ -84,5 +86,29 @@
         <INDEX NAME="ctx-timecreated" UNIQUE="false" FIELDS="timecreated"/>
       </INDEXES>
     </TABLE>
+    <TABLE NAME="block_opencast_catalog" COMMENT="this table contains the list of metadata catalog and its data types">
+      <FIELDS>
+        <FIELD NAME="id" TYPE="int" LENGTH="10" NOTNULL="true" SEQUENCE="true"/>
+        <FIELD NAME="name" TYPE="char" LENGTH="255" NOTNULL="true" SEQUENCE="false"/>
+        <FIELD NAME="datatype" TYPE="char" LENGTH="255" NOTNULL="true" SEQUENCE="false"/>
+        <FIELD NAME="required" TYPE="int" LENGTH="1" NOTNULL="true" DEFAULT="0" SEQUENCE="false"/>
+        <FIELD NAME="readonly" TYPE="int" LENGTH="1" NOTNULL="true" DEFAULT="0" SEQUENCE="false"/>
+        <FIELD NAME="default_json" TYPE="text" NOTNULL="false" SEQUENCE="false"/>
```

Comment:
You have inconsistencies in the naming of this column. In the code you often name it param_json, which leads to errors when executing the code.